packages/eslint-plugin/src/configs/recommended.json

```diff
@@ -2,35 +2,41 @@
   "extends": "./configs/base.json",
   "rules": {
     "@typescript-eslint/adjacent-overload-signatures": "error",
-    "@typescript-eslint/array-type": "error",
+    "@typescript-eslint/await-thenable": "error",
+    "@typescript-eslint/ban-ts-ignore": "error",
     "@typescript-eslint/ban-types": "error",
     "camelcase": "off",
     "@typescript-eslint/camelcase": "error",
     "@typescript-eslint/class-name-casing": "error",
-    "@typescript-eslint/consistent-type-definitions": "error",
     "@typescript-eslint/explicit-function-return-type": "warn",
-    "@typescript-eslint/explicit-member-accessibility": "error",
-    "indent": "off",
-    "@typescript-eslint/indent": "error",
     "@typescript-eslint/interface-name-prefix": "error",
     "@typescript-eslint/member-delimiter-style": "error",
     "@typescript-eslint/no-angle-bracket-type-assertion": "error",
     "no-array-constructor": "off",
     "@typescript-eslint/no-array-constructor": "error",
+    "@typescript-eslint/no-empty-function": "error",
```

Comment:
Should we disable [no-empty-function](https://eslint.org/docs/rules/no-empty-function) rule?

Reply:
I see it in the same vein as `no-empty-interface`.
It's a best practice to avoid having useless code.

I don't see why you wouldn't want this rule turned on, unless you're purposely defining a "noop" function.
But I would see it as a good thing, as you'd have to comment explaining why the empty function exists.

Reply:
My intention was that, should we disable `no-empty-function` because we enabled `@typescript-eslint/no-empty-function` if the two are duplicated?

Reply:
I'm sorry! I completely misunderstood.

Yes, we should. Definitely. I completely forgot about the base rules!